### PR TITLE
text finished

### PR DESCRIPTION
**Pull request by bird926**

### CODE REVIEW

File: docs/bin/map/object/test/text.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <script charset="utf-8" src="https://map.qq.com/api/js?v=2.exp&key=DDNBZ-VUNKF-VVJJA-NSN2B-66R5H-VFF23"></script>
+
+</head>
+
+<body onLoad="init()">
+<button id="visible-text" type="button" class="btn btn-default btn-sm">显示全量文本信息</button>
+<script>
+    function init() {
+        const Text = class {
+            constructor(map, elementId="visible-text") {
```
Comment (evanwangxx) [inline docs/bin/map/object/test/text.html]: put elementId within L27

File: docs/bin/map/object/test/text.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                let textBlock = new qq.maps.Label({
+                    position: center,
+                    map: this.map,
+                    content: text,
+                    zIndex: this.height
+                });
+
+                textBlock.setStyle(cssP);
+
+                try{
```
Comment (evanwangxx) [inline docs/bin/map/object/test/text.html]: format

File: docs/bin/map/object/text.js [hunk 3]
```diff
@@ -2,3 +2,46 @@
...
+		let textBlock = new qq.maps.Label({
+			position: center,
+			map: this.map,
+			content: text,
+			zIndex: this.height
+		});
+
+		textBlock.setStyle(cssP);
+
+		try{
```
Comment (evanwangxx) [inline docs/bin/map/object/text.js]: format

File: docs/bin/map/object/text.js [hunk 4]
```diff
@@ -2,3 +2,46 @@
 // map-text library for TerraEyes
 // (c) 2019 Hongbo Wang
 // Copyright © 1998 - 2019 Tencent. All Rights Reserved.
+
+const Text = class {
+	constructor(map, elementId="visible-text") {
```
Comment (evanwangxx) [inline docs/bin/map/object/text.js]: put elementId in L27

File: docs/bin/map/object/text.js [hunk 5]
```diff
@@ -2,3 +2,46 @@
 // map-text library for TerraEyes
 // (c) 2019 Hongbo Wang
```
Comment (evanwangxx) [inline docs/bin/map/object/text.js]: author name

**Review by evanwangxx — CHANGES_REQUESTED** [review]

**Comment by bird926** [comment]
text finish, bug fixed

**Review by evanwangxx — APPROVED** [review]

**Pull request merged**